src/pages/Beta/Pool/index.tsx

```diff
@@ -155,6 +173,8 @@ const PoolUI = () => {
     [setMenu]
   )
 
+  const version = activeMenu === MenuType.allFarmV1 ? 1 : activeMenu === MenuType.allFarmV3 ? 3 : 2
+
```

Comment:
can we please use `ChefType` instead of `1, 2, 3`. I have added same comment to components PR also to make sure we cover it.

Reply:
Yes, I ended up letting this slip when I copied the files